qbft/instance.go

```diff
@@ -24,6 +24,7 @@ type Instance struct {
 	// forceStop will force stop the instance if set to true
 	forceStop  bool
 	StartValue []byte
+	CdFetcher  *types.DataFetcher `json:"-"`
```

Comment:
Could we rename CdFetcher to CDFetcher for clarity? 'CD' stands for 'Consensus Data', and capitalizing the acronym could improve readability and align with our naming conventions.

Reply:
I agree. I even prefer the whole `ConsensusDataFetcher`, or simply `DataFetcher`.

Reply:
Lets to this, I prefer `DataFetcher` or `Fetcher`. anyway if using acronym always capital letters for all words.